backstage.json

```diff
@@ -1,3 +1,3 @@
 {
-  "version": "1.23.4"
+  "version": "1.25.2"
```

Comment:
hoping that this change will cascade automatically to the build metadata file... if not we can revisit https://issues.redhat.com/browse/RHIDP-1414